mappings/net/minecraft/server/world/ServerWorld.mapping

```diff
@@ -55,3 +60,4 @@ CLASS uq net/minecraft/server/world/ServerWorld
 	METHOD v getFluidTickScheduler ()Lbce;
 	METHOD w getServer ()Lnet/minecraft/server/MinecraftServer;
 	METHOD x getEntityTracker ()Lul;
+	METHOD y getPortalForcer ()Lbcc;
```

Comment:
PortalForcer?

Reply:
It's a vanilla name, as shown by the profiler section name. It is also a name of a vanilla [MCPE class](https://gist.githubusercontent.com/Runemoro/518a78eb997346a31242b1e80d7da19c/raw/d04af659b8463f77718b437ea81bce5d2b5d7907/mcpe-1.9.0.3.txt).